overviewer_core/world.py

```diff
@@ -792,9 +792,16 @@ def get_worlds():
     for dir in os.listdir("."):
         world_dat = os.path.join(dir, "level.dat")
         if not os.path.exists(world_dat): continue
-        info = nbt.load(world_dat)[1]
-        info['Data']['path'] = os.path.join(".", dir).decode(loc)
-        if 'LevelName' in info['Data'].keys():
-            ret[info['Data']['LevelName']] = info['Data']
+        world_path = os.path.join(".", dir)
+        try:
```

Comment:
Isn't this logic also needed on [line 786](https://github.com/overviewer/Minecraft-Overviewer/pull/1354/files#diff-225673b5d5eed8a35e29b5f470e04e6aL786) above?